1.8/usage/service-discovery/dns-overview.md

```diff
@@ -0,0 +1,65 @@
+---
+post_title: DNS Quick Reference
+nav_title: DNS Reference
+menu_order: 20
+---
+
+DC/OS offers a lot when it comes to DNS.
+This quick reference provides a summary of the available options.
+
+# Example Setup
+
+To make things more concrete, we'll use this example application:
+* (Group) `outergroup` > (Group) `subgroup` > (Service Name) `myapp`
+* Port: `555`
+ * Port Name: `myport`
+ * Load Balanced
+* Running on a Virtual Network
+* Running on the Marathon framework (if unsure, it's probably this one)
+ * If you are running another framework, then replace any instance of
+   `marathon` with the name of your framework.
+
+On this example setup:
+* Default DC/OS Name (if unsure, it's probably this one)
```

Comment:
Replace lines 22 - 25 with: DC/OS uses `thisdcos` as the default DC/OS name. If you provided a custom DC/OS name, replace any instance of `thisdcos` with the name that you provided.